src/ui/header/localeButton/LocaleButton.tsx

```diff
@@ -0,0 +1,15 @@
+import { LocaleButtonProps } from './LocaleButton.types';
+import * as Styled from './LocaleButton.styles';
+import { useLocale } from 'hooks';
+
+export const LocaleButton = ({ text, onClick }: LocaleButtonProps) => {
+  const { locale } = useLocale();
```

Comment:
Zauważ, że komponent `LocaleButton` jest w tym przypadku zależny od zewnętrznego hooka. Jeśli chciałbyś go przetestować, czy dobrze wyświetla się `active`/`not-active`, wtedy musiałbyś mocować całego hooka.
Lepszym rozwiązaniem byłoby, żeby komponent dostawał w propsach tę wartość jako `isActive` (bo jego parent i tak wywołuje `useLocale`)